### ISSUE TITLE
iOS?

### ISSUE THREAD

**Issue by itdelatrisu** [fluddokt/opsu#19]
Just wondering, has anyone tried to build this for iOS?  Are there any reasons why it wouldn't work?  (I don't have a Mac, so I can't try it.)

**Comment by Reddy360** [comment]
This is programmed in Java, iOS apps are programmed in Objective C and Swift.

It will require a full port to work.

**Comment by itdelatrisu** [comment]
You can deploy libgdx apps to iOS.  [See here.](https://github.com/libgdx/libgdx/wiki/Deploying-your-application#deploy-to-ios)

**Comment by abalabahaha** [comment]
But even if it was ported, how would it be distributed?

**Comment by itdelatrisu** [comment]
I don't know much about iOS development, but it's possible to install an app on a connected iOS device from XCode, right?  At the very least, people could build the app themselves, then.  I'm more interested in whether or not it would work at all, and how much code actually needs to be changed...

**Comment by abalabahaha** [comment]
Yeah you can load an app onto a connected device, and I think you can do that without an Apple Developer account now. I might attempt to screw around later if I get the time.
